### PR TITLE
Add error handling for ns creation

### PR DESCRIPTION
**Pull request by wg102**
PR for https://jirab.statcan.ca/browse/BTIS-99

### CODE REVIEW

**Review by bryanpaget — APPROVED** [review]
lgtm

File: components/centraldashboard/public/components/landing-page.js [hunk 1]
```diff
@@ -106,37 +106,22 @@ export class LandingPage extends mixinBehaviors([AppLocalizeBehavior], utilities
...
-        }
+
         /*
          * Poll for profile over a span of 20 seconds (every 300ms)
          * if still not there, let the user click next again!
          */
         const success = await this.pollProfile(66, 300);
         if (success) this._successSetup();
 
+        // console.log('Trigger second part');
```
Comment (mathis-marcotte) [inline components/centraldashboard/public/components/landing-page.js]: maybe just delete this commented out console.log

File: components/centraldashboard/public/components/landing-page.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```
Comment (mathis-marcotte) [inline components/centraldashboard/public/components/landing-page.js]: I think this was added for debugging (I know because i do it too)

**Review by mathis-marcotte — CHANGES_REQUESTED** [review]

**Review by mathis-marcotte — APPROVED** [review]
lgtm

**Pull request merged**